src/route-href.js

```diff
@@ -0,0 +1,40 @@
+import {customAttribute, bindable} from 'aurelia-templating';
+import {inject} from 'aurelia-dependency-injection';
+import {Router} from 'aurelia-router';
+import {TaskQueue} from 'aurelia-task-queue';
```

Comment:
This is just to coalesce changes from the 3 bindable change handlers that fire with initial values.

Are the 'Dependencies' and 'Used by' README sections manually maintained?  If so we'll need to update this repo, and aurelia/task-queue.